packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-sign-up/confirm-sign-up.component.ts

```diff
@@ -37,13 +42,28 @@ export class ConfirmSignUpComponent implements OnInit {
   }
 
   public setFormFields() {
-    const _state = this.authenticator.authState;
-    this.formOverrides =
-      getActorState(_state).context?.formFields?.confirmSignUp;
-  }
+    /**
+     * @todo(breaking): Angular has different placholder here from other frameworks.
+     *
+     * Translating here in a backwards-compatible manner, but should be resolved in next major version.
+     */
```

Comment:
So if they have not translated the text then show them the new placeholder, otherwise show them the older one? And this `hasTranslations` detects any user translations?

Reply:
`hasTranslations` detects only user translations. So yes, if they have no translations at all, it'll show the new placeholder. If they have translation for the new placeholder, we will prioritize that. If they have translation for the old placeholder but not for the new one, we'll translate theirs.